qt/app.py

```diff
@@ -799,7 +799,7 @@ def _create_main_toolbar(self):
 
                 toolbar.widgetForAction(act).setToolTip(button_tip)
 
-            act.setText(TextWrapper(width=8, break_long_words=False).fill(act.text()))
+            act.setText(fill(act.text(), width=8, break_long_words=False))
```

Comment:
The term `fill` is to broad. I would like to keep the package name to make the context clear to the readers.
```suggestion
            act.setText(textwrap.fill(act.text(), width=8, break_long_words=False))
```